dev_guide/app_tutorials/jenkins_pipeline.adoc

```diff
@@ -0,0 +1,185 @@
+[[dev-guide-jenkins-pipeline-builds]]
+= Jenkins Pipeline Builds
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[introduction]]
+
+== Introduction
```

Comment:
This is about openshift pipelines first and foremost and the jenkins openshift client plugin secondarily, so the overview should start by talking about openshift pipelines, before jumping into the client plugin.

(you can have an openshift pipeline that doesn't use the client plugin)

Reply:
Updated in next push